src/client/client.h

```diff
@@ -44,32 +42,33 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #define CLIENT_CHAT_MESSAGE_LIMIT_PER_10S 10.0f
 
-struct ClientEvent;
-struct MeshMakeData;
-struct ChatMessage;
-class MapBlockMesh;
-class RenderingEngine;
-class IWritableTextureSource;
-class IWritableShaderSource;
-class IWritableItemDefManager;
+class Camera;
+class ClientMediaDownloader;
 class ISoundManager;
+class IWritableItemDefManager;
+class IWritableShaderSource;
+class IWritableTextureSource;
+class MapBlockMesh;
+class MapDatabase;
+class MeshUpdateManager;
+class Minimap;
+class ModChannelMgr;
+class MtEventManager;
+class NetworkPacket;
 class NodeDefManager;
-//class IWritableCraftDefManager;
-class ClientMediaDownloader;
+class ParticleManager;
+class RenderingEngine;
 class SingleMediaDownloader;
+struct ChatMessage;
+struct ClientDynamicInfo;
+struct ClientEvent;
 struct MapDrawControl;
-class ModChannelMgr;
-class MtEventManager;
-struct PointedThing;
 struct MapNode;
-class MapDatabase;
-class Minimap;
+struct MeshMakeData;
 struct MinimapMapblock;
-class MeshUpdateManager;
-class ParticleManager;
-class Camera;
 struct PlayerControl;
-class NetworkPacket;
+struct PointedThing;
```

Comment:
Note to reviewers: alphabetical sorting. `ClientDynamicInfo` was added, nothing removed.